src/custom_nodes/object_detection/README.md

```diff
@@ -0,0 +1,62 @@
+# Custom node for object detection models from OpenVINO Model Zoo
+
+This custom node analyses the response of models from OpenVINO Model Zoo. Based on the inference results and the original image,
+it generates a list of detected boxes for following object recognition models. 
+Each image in the output will be resized to the predefined target size to fit the next inference model in the 
+DAG pipeline.
+Additionally to the detected text boxes, in the two additional outputs are returned their coordinates with information about coordinates
+and confidence levels for the filtered list of detections.
+
+# Supported models
+
+All [OpenVINO Model Zoo](!https://github.com/openvinotoolkit/open_model_zoo/tree/master/models/intel) object detection models with specific output tensor `detection` with shape: `[1, 1, 200, 7]`:
+- face-detection
+- face-detection-adas
+- face-detection-retail
+- person-detection
+- person-detection-adas
+- person-detection-retail
+- vehicle-detection
+- vehicle-detection-adas
+- product-detection
+- person-vehicle-bike-detection
+- vehicle-license-plate-detection
+- pedestrian-and-vehicle-detector
+
+# Building custom node library
+
+You can build the shared library of the custom node simply by running command in this custom node folder context:
+```
+make
+```
+It will compile the library inside a docker container and save the results in `lib` folder.
+
+# Custom node inputs
+
+| Input name       | Description           | Shape | Precision |
+| ------------- |:-------------:| -----:| -----:|
+| image      | Input image in an array format. Only batch size 1 is supported and images must have 3 channels. Resolution is configurable via parameters `original_image_width` and `original_image_height` | `1,3,H,W` | FP32 |
+| detection      | object detection model output | `1,1,200,7` | FP32 |
+
+
+# Custom node outputs
+
+| Output name        | Description           | Shape  | Precision |
+| ------------- |:-------------:| -----:| -----:|
+| images      | Returns images representing detected text boxes. Boxes are filtered based on confidence_threshold param. Resolution is defined by the node parameters. All images are in a single batch. Batch size depend on the number of detected objects.  | `N,1,C,H,W` | FP32 |
+| coordinates      | For every detected box `N` the following info is added: x coordinate for the box center, y coordinate for the box center, box original width, box original height | `N,1,4` | I32 |
+| confidences |   For every detected box `N` information about confidence level | `N,1,1` | FP32 |
```

Comment:
I guess this `N` has something to do with demultiplexing? Can we link explanation here?